### DISCUSSION TITLE
ml5.js / vite - webcam activated but not showing

### DISCUSSION THREAD

**Discussion by laura-nguyen** [community/community#129452] [Programming Help]
### Body

Hi! I'm trying to classify a yoga pose via webcam, and I already trained the model. Unfortunately, the webcam isn't showing up even though it's on. It is showing up as a black box on the browser. I was wondering if anyone could take a look at my repo and see if there's anything wrong with it. this is my repo: https://github.com/laura-nguyen/yoga-ai-test

Thanks!
Laura 

### Guidelines

- [X] I have read and understood this category's [guidelines](https://github.com/orgs/community/discussions/51384) before making this post.

**Comment by ebndev** [comment]
Hi @laura-nguyen, thanks for being a part of the GitHub Community!

Just wanted to say this sounds like an awesome project 😄
